### DISCUSSION TITLE
Scrape errors when initiate main.py

### DISCUSSION THREAD

**Discussion by Armstock** [omkarcloud/google-maps-scraper#143] [General]
I'm really sure about my local network and dependencies is ok, but it always told me scraper can not fetch certain informations for me, please check my screenshot below: 
![screenshot-20240313-152010](https://github.com/omkarcloud/google-maps-scraper/assets/56221017/01352594-36b4-460c-81ad-fb77f463cfcf)
Can developer give me some hints?

**Comment by Chetan11-dev** [comment]
We have released new version, which should resolve these errors, Let me know if it reoccurs?
